src/jquery.mockjax.js

```diff
@@ -269,6 +269,7 @@
 				url: mockHandler.proxy,
 				type: mockHandler.proxyType,
 				data: mockHandler.data,
+				async: false,
```

Comment:
this will not change the nature of the mocked request. In this implementation proxy data will always be loaded as a synchronous call(if specifies). It will not change the synchronous or the asynchronous nature of the original mocked call.

Compared to the previous implementation if we have a **proxy** with a **synchronous ajax** it simply **does not work**. This will remove that.